distributed/tests/test_client.py

```diff
@@ -5819,22 +5819,6 @@ async def test_client_timeout_2():
         assert stop - start < 1
 
 
-@gen_test()
-async def test_client_active_bad_port():
```

Comment:
http servers wait for a `\r\n` before parsing the request line and so we never get a handshake back from those servers so this test fails